blueprints/component/module-unification-files/__root__/__path__/component.ts

```diff
@@ -1,7 +1,5 @@
 import Component from '@ember/component';
 
-export default class <%= classifiedModuleName %> extends Component.extend({
-  // anything which *must* be merged to prototype here
-}) {
-  // normal class body definition here
+export default class <%= classifiedModuleName %> extends Component {
```

Comment:
Let’s delete module unification files as well (again, doesn’t have to be this PR). Module unification is dead, long live template imports. 😅

Reply:
Make sure everyone who's using module unification has stopped using it before doing that...

Reply:
Or throw an error with a link to a codemod that migrates the old fashioned folk up to an octane layout

Reply:
While that’d be a nice thing to do I don’t think we *need* to do it. When we do it, we will cut a major release and note that we’ve dropped support for MU. Ember itself stopped supporting it in the blueprints long since, I believe, and we don’t want to support generating *more* code that way from our POV. We already dropped support for it in a variety of places in the core addon.